### DISCUSSION TITLE
Console ports for 3DS or even NDS

### DISCUSSION THREAD

**Discussion by AppTesterMC** [ihhub/fheroes2#6967] [Ideas]
I was thinking if this is even a viable endeavor to port fHeroes2 to 3DS console as I think this handheld have enough processing power to handle it, even NDS (Nintendo DS) might be powerful enough to handle the game. I was thinking that even maybe a split on the top screen the big map/overwolrd and on the touchscreen the side panel, mini map, controls, menus would be useful!

Am I missing something here? Is there maybe a reason that we cannot consider such a port?

Thank you for your feedback and insight on this idea.

**Comment by ihhub** [comment]
Hi @AppTesterMC , theoretically it is possible but practically it is not. Screen resolution is too small and adopting the engine for a very specific lower than the original resolution device is not worth it. If someone wants to do such a port they are more than welcome but for now this is not in the scope of this project.

**Reply by AppTesterMC** [reply]
Thank you for answering so quickly!

Ok I see your point. NDS has two screens with resolution each 256 x 192 pixels. and 3DS has top screen with (800 x 240) or 400 x 240 (?) and bottom touch screen with 320x240 while the smallest fHeroes2 resolution is 640 x 480. It will require some trickery to get this to work.

Well that's why I asked to get some insight!
